diff --git a/node_modules/pocketbase-react/es/context/auth.d.ts b/node_modules/pocketbase-react/es/context/auth.d.ts
index 19c90d7..1bcc7d1 100644
--- a/node_modules/pocketbase-react/es/context/auth.d.ts
+++ b/node_modules/pocketbase-react/es/context/auth.d.ts
@@ -28,6 +28,7 @@ export interface AuthActions {
     updateProfile: UpdateProfileType;
     updateEmail: UpdateEmailType;
     deleteUser: DeleteUserType;
+    authProviders: AuthProviderInfo[]|undefined;
 }
 export declare const AuthContext: React.Context<AuthActions>;
 export type AuthProviderProps = {
diff --git a/node_modules/pocketbase-react/es/context/auth.js b/node_modules/pocketbase-react/es/context/auth.js
index 1e27c72..d8a34b9 100644
--- a/node_modules/pocketbase-react/es/context/auth.js
+++ b/node_modules/pocketbase-react/es/context/auth.js
@@ -54,7 +54,7 @@ export const AuthProvider = props => {
     },
     deleteUser: async id => {
       await (client == null ? void 0 : client.collection('users').delete(id));
-    }
+    },
   };
   React.useEffect(() => {
     (async () => {
@@ -63,6 +63,6 @@ export const AuthProvider = props => {
     })();
   }, [props.webRedirectUrl, props.mobileRedirectUrl]);
   return /*#__PURE__*/React.createElement(AuthContext.Provider, {
-    value: actions
+    value: {...actions, authProviders}
   }, props.children);
 };
\ No newline at end of file
